tests/monster_test.fbs

```diff
@@ -6,6 +6,11 @@ namespace MyGame.Example2;
 
 table Monster {}  // Test having same name as below, but in different namespace.
 
+/// This will test a potential "ambiguous call to overloaded function" using the same data from different namespace.
+table MonsterStatus {
```

Comment:
Don't know how from this clunky web interface! Why is it so hard to find and manage your commits?